app/transforms/file.js

```diff
@@ -1,6 +1,5 @@
 import Ember from 'ember';
 import DS from 'ember-data';
-import getOwner from 'ember-getowner-polyfill';
```

Comment:
Scrap that, it should remain in the package to polyfill for older ember versions